bind/gencpy.go

```diff
@@ -243,11 +243,17 @@ func (g *cpyGen) gen() error {
 
 	for _, v := range g.pkg.vars {
 		name := v.Name()
+		get := v.doc
+		set := v.doc
+		if v.doc == "" {
```

Comment:
@sbinet 
Looks like this is duplicated part of this [code](https://github.com/go-python/gopy/blob/a4a47725eb8640cfbf40aca10dbf712c392b4fab/bind/gencpy.go#L319).
Is there no way to use this part?